fix: articles/active-directory/manage-apps/use-scim-to-provision-users-and-groups.md

nschonni opened the pull request:
- Double word "for"
- Add code fence language
- 4 backticks to 3
- Unclosed code fences

ankitC commented on articles/active-directory/manage-apps/use-scim-to-provision-users-and-groups.md:
````diff
@@ -823,7 +827,7 @@ Developers using the CLI libraries provided by Microsoft for building a SCIM ser
 
-   ```
+   ```csharp
      public override Action\<Owin.IAppBuilder, System.Web.Http.HttpConfiguration.HttpConfiguration\> StartupBehavior
````
Action<Owin.IAppBuilder, System.Web.Http.HttpConfiguration.HttpConfiguration> StartupBehavior

nschonni replied:
Good catch, fixed

ankitC reviewed (changes requested):
Thanks for the changes.

nschonni committed:
fix: articles/active-directory/manage-apps/use-scim-to-provision-users-and-groups.md

- Double word "for"
- Add code fence language
- 4 backticks to 3
- Unclosed code fences
- Unsecape angle bracket in code fence
